chia/simulator/setup_services.py

```diff
@@ -105,6 +105,7 @@ async def setup_full_node(
     config = local_bt.config
     service_config = config["full_node"]
     service_config["database_path"] = db_name
+    service_config["testing"] = True
```

Comment:
these changes look unrelated. Are they deliberate? if so, can you explain why you need to make this change?

Reply:
deliberate otherwise we throw an error here https://github.com/Chia-Network/chia-blockchain/blob/a9b3e25d2ae037e29a0acd7b0fae7450e139e087/chia/server/server.py#L688

this is whole thing is done in a bad way but im not going to fix that 
here, i can set the config inside the test itself if you think its better but i figured id save other people the trouble.